ciongke/cmd/hook/deployment.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: hook
+  labels:
+    app: hook
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        app: hook
+    spec:
+      terminationGracePeriodSeconds: 30
+      containers:
+      - name: hook
+        image: "gcr.io/k8s-ci-on-gke/hook:0.0"
```

Comment:
I want the image of a deployment to be `gcr.io/$(PROJECT)/hook:$(VERSION)`. It seems like there's not a good way to do this with a configmap, so I have to either use sed and bash or just embed the yaml in a makefile. Neither are beautiful, but it's not too bad.